UI/Web/src/app/manga-reader/_components/single-renderer/single-renderer.component.ts

```diff
@@ -67,6 +74,28 @@ export class SingleRendererComponent implements OnInit, ImageRenderer {
       takeUntilDestroyed(this.destroyRef)
     );
 
+    //handle manual width
+    this.mangaReaderComponent.generalSettingsForm.get("widthSlider")?.valueChanges.pipe(takeUntilDestroyed(this.destroyRef)).subscribe(val => {
+      val = (val < 0) ? 'none' : val + "%";
+      this.widthOverride$ = val;
+
+      var img = this.document.getElementById('image-1');
+      if(!img) return;
+
+      var scaling = this.mangaReaderComponent.generalSettingsForm.get("fittingOption")?.value;
+      if (FITTING_OPTION.HEIGHT == scaling) return;
+
+      if(this.widthOverride$ == 'none') img.style.width = "";
+      else img.style.width = this.widthOverride$;
+    });
+    //reset manual width when fitting option changes
+    this.mangaReaderComponent.generalSettingsForm.get("fittingOption")?.valueChanges.pipe(takeUntilDestroyed(this.destroyRef)).subscribe(val => {
```

Comment:
This isn't how you should code it. Don't pass the parent component into the child. You can get fittingOption from readerSettings$ and if you can't, update the code so it's packed in the ReaderSetting interface.

Reply:
so from the mangareader component updating the setting and in the renderers subscribing to the settings value to make changes?

Reply:
Yes, see imageFitClass$ in the double-renderer and my comment below.